### PR TITLE
CLDSRV-209: allow to run cloudserver with empty details for dmf

### PR DESCRIPTION
**Pull request by ghost**
Fixes Zenko until dmf is added in ZKOP.

### CODE REVIEW

**Review by XinLi0207 — APPROVED** [review]

File: lib/Config.js [hunk 1]
```diff
@@ -192,32 +192,34 @@ function hdClientLocationConstraintAssert(configHd) {
...
-    assert.strictEqual(Array.isArray(repoId));
-    repoId.forEach(rId => {
-        assert(typeof rId === 'string',
-            `bad config: ${rId} must be a string`);
-    });
-    assert(repoId.every(
-        r => typeof r === 'string'),
-        'bad config: each repoId must be a string',
-    );
+    if (Object.keys(locationObj.details).length > 0) {
```
Comment (miniscruff) [inline lib/Config.js]: I would switch to a guard clause, no need for the extra tabbing and easier to remove later
```suggestion
    if (Object.keys(locationObj.details).length === 0) {
        return
    }
```

Comment (ghost) [reply lib/Config.js]: Agreed, thanks as always :smile:

Comment (ghost) [reply lib/Config.js]: Done: https://github.com/scality/cloudserver/pull/4543/commits/78498030ab902107a088602c798f28653c81f98c

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**